Cargo.toml

```diff
@@ -1,17 +1,21 @@
 [package]
 name         = "cryptobox-c"
-version      = "1.0.0"
+version      = "2.0.0"
```

Comment:
Perhaps let's not bump the version (unless there's a good reason to?)

Reply:
ok